package.json

```diff
@@ -3,18 +3,30 @@
   "version": "2.0.0",
   "main": "./src/customfunctions.js",
   "scripts": {
-    "start": "start npm run dev-server && start npm run watch && npm run sideload",
+    "start": "start npm run packager && start npm run dev-server && start npm run watch && npm run sideload",
```

Comment:
Please update this based on excel-functions in fabric-internal.

Reply:
Should we have "start" only do the devserver and watch?  (esp. as our recommendation, for now, is to always start with just getting it to work on the web)?  I.e., replace "start" with what we have for "start-web".

And then have start-desktop do the sideloading (or keep them as separate commands, that's ok too).